mifospay/src/main/res/layout/fragment_merchants.xml

```diff
@@ -12,5 +12,29 @@
         android:layout_gravity="center"
         android:layout_margin="@dimen/marginItemsInSectionLarge"
         android:visibility="gone" />
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:id="@+id/merchant_fragment_layout"
+        android:layout_height="match_parent"
+        android:visibility="invisible"
```

Comment:
Change visibility to "gone" instead of "invisible"

Here's why https://stackoverflow.com/questions/11556607/android-difference-between-invisible-and-gone